tools/setup_discord.py

```
@@ -0,0 +1,414 @@
+    ensure_channel(guild_id, token, all_channels, "dev-log", cat_dev,
+                   "Brief notes from the maintainer when notable fixes land.",
+                   ow_devlog)
+
+    # -----------------------------------------------------------------------
+    # Done
+    # -----------------------------------------------------------------------
+    print("\n" + "=" * 60)
+    print("Setup complete!")
+    print()
+    print("Next steps:")
+    print(f"  1. Assign @Developer to yourself:")
```

Ruff (0.15.12)

[error] f-string without any placeholders

Remove extraneous `f` prefix

(F541)